.github/uffizzi/uffizzi.production.app-config.yaml

```diff
@@ -0,0 +1,10 @@
+app:
+  title: Backstage Uffizzi Environment
+  baseUrl: ${UFFIZZI_URL}
+
+backend:
+  baseUrl: ${UFFIZZI_URL}
+  cors:
+    origin: ${UFFIZZI_URL}
+    methods: [GET, POST, PUT, DELETE]
+    credentials: true
```

Comment:
This one doesn't have any catalog data. In a future iteration (not necessary for this first PR) we should probably make it have some of that locally in the image at least (maybe the example data that's already in `packages/catalog-model/examples`?), so we can get more of a rich testing experience.

Reply:
I will keep this for now and update this change in a subsequent PR.